src/porepy/models/poromechanics.py

```diff
@@ -71,6 +71,20 @@ class EquationsPoromechanics(
 ):
     """Combines mass and momentum balance and contact mechanics equations."""
 
+    def body_force(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
```

Comment:
Have you considered THM?

Reply:
You are the expert. I am not to well familiar with THM modeling. But in the end, it is still fluid and solid, i.e., the bulk, on which gravity acts. Temperature implicitly enters through temperature dependent porosity and density laws. Is there anything other to consider?

Reply:
I would guess @IvarStefansson's point is that a similar change would be needed in the THM model.

Reply:
Yes, that was what I was thinking. Unless THM inherits this change from HM.

Reply:
Indeed. I did not realize that pp.ThermoPoromechanics does not inherit its equations from pp.Poromechanics. I made the same adaptation. BTW, I just used the Equations class for overwriting what has been previously defined in MomentumBalanceEquations. If you think this is the wrong place, feel free to suggest alternatives.